src/ontology/uberon-edit.obo

```diff
@@ -223988,7 +223962,6 @@ id: distalmost_part_of
 name: distalmost_part_of
 def: "X distalmost_part_of Y <=> X is part_of Y and X is adjacent_to the distal boundary of Y." [https://orcid.org/0000-0002-6601-2165]
```

Comment:
delte def?

Reply:
All the definitions I didn't delete are different in the import files. Maybe it's an updated version?

Reply:
If there is a definition present in the import at all, delete this one here. If there is not, make an issue at repo of imported ontology!